src/abstractdataframe/abstractdataframe.jl

```diff
@@ -288,38 +288,33 @@ Base.isempty(df::AbstractDataFrame) = size(df, 1) == 0 || size(df, 2) == 0
 ##
 ##############################################################################
 
-head(df::AbstractDataFrame, r::Int) = df[1:min(r,nrow(df)), :]
-head(df::AbstractDataFrame) = head(df, 6)
-tail(df::AbstractDataFrame, r::Int) = df[max(1,nrow(df)-r+1):nrow(df), :]
-tail(df::AbstractDataFrame) = tail(df, 6)
-
 """
-Show the first or last part of an AbstractDataFrame
+    first(df::AbstractDataFrame)
 
-```julia
-head(df::AbstractDataFrame, r::Int = 6)
-tail(df::AbstractDataFrame, r::Int = 6)
-```
-
-**Arguments**
+Get the first row of `df` as a `DataFrameRow`.
+"""
+Base.first(df::AbstractDataFrame) = df[1, :]
 
-* `df` : the AbstractDataFrame
-* `r` : the number of rows to show
+"""
+    first(df::AbstractDataFrame, n::Integer)
 
-**Result**
+Get a data frame with the `n` first rows of `df`.
+"""
+Base.first(df::AbstractDataFrame, n::Integer) = df[1:min(n,nrow(df)), :]
```

Comment:
Maybe it would be better to be a bit breaking and return a `SubDataFrame`? This would be consistent with `first(df)` which also returns a view now. I do not have very strong preference here over `DataFrame` so @nalimilan - please decide.

Reply:
Good question. `first(df)` returns a view only because `df[1, :]` does, so I'd say it's consistent to also use `df[1:i, :]` here, and therefore return a copy.